recipes/contact_map/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "contact_map" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "695967007dc43b639efd458130411463ce79893863699d2d32b60c1b848cd062" %}
+# sha256 comes from 
+# openssl sha256 dist/contact_map-{{ version }}.tar.gz
+# (after `python setup.py sdist`)
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - future
+    - numpy
+    - mdtraj
+    - pandas
+  run:
+    - python
+    - future
+    - numpy
+    - mdtraj
+    - pandas
+
+test:
+  imports:
+    - contact_map
+
+about:
+  home: http://github.com/dwhswenson/contact_map
+  license: LGPL-2.1+
+  license_family: LGPL
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: LICENSE.txt
```

Comment:
It doesn't look like the license is in the upstream `MANIFEST.in`, so it's not being packaged with the source distribution. You can manually package it by adding a copy of the license to this directory and then:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
Thanks -- the better approach would be to update the `MANIFEST.in`, right? 

This 0.1.x series is mainly for me to play with getting distribution/deployment set up, so I'm just as happy to bump to 0.1.2 on PyPI (including adding `LICENSE` to the sdist) and have that be the first conda-forge release.

Other changes have been implemented already; I'll push those when I have a release that includes the license file on PyPI.